internal/pkg/image/image.go

```diff
@@ -18,6 +18,7 @@ const (
 	CSIv0ExternalProvisionerContainerImage    = "storageos/csi-provisioner:v0.4.3"
 	CSIv0ExternalAttacherContainerImage       = "quay.io/k8scsi/csi-attacher:v0.4.2"
 	DefaultNFSContainerImage                  = "storageos/nfs:1.0.0"
+	DefaultAPIManagerImage                    = "storageos/api-manager:v0.1.0"
```

Comment:
Maybe set it to `develop` tag for now, until we have a versioned release. We can update this along with the node container before release.

Reply:
Pushed `storageos/api-manager:develop`